assets/scripts/info_bubble/BuildingHeightControl.jsx

```diff
@@ -100,10 +88,21 @@ function mapStateToProps (state, ownProps) {
   }
 }
 
-const actionCreators = {
+const mapDispatchToProps = {
   addBuildingFloor,
   removeBuildingFloor,
   setBuildingFloorValue
 }
 
-export default injectIntl(connect(mapStateToProps, actionCreators)(BuildingHeightControl))
+BuildingHeightControl.propTypes = {
```

Comment:
Let's move `propTypes` above the `BuildingHeightControl` declaration (so it's in a similar position in the file compared to a class component). You'll have to convert the `const` to `function` so it's hoisted.

Reply:
Corrected.